pep-0613.rst

```diff
@@ -88,7 +88,7 @@ across the codebase.
 
 With explicit aliases, the type checker has enough information to error on the
 actual definition of the bad type alias, and explain why: that :code:`MyGeneric(int)`
-and `InvalidType` are not valid types. When the value expression is no longer
+and code:`InvalidType` are not valid types. When the value expression is no longer
```

Comment:
The `code:` prefix is used here for consistency because the rest of this file uses it.

Reply:
I guess you'd like to use `:code:` role here, right?

Reply:
I believe @tk0miya is suggesting:
```suggestion
and :code:`InvalidType` are not valid types. When the value expression is no longer
```
But we are not using docutils, so it probably should be:
```suggestion
and ``InvalidType`` are not valid types. When the value expression is no longer
```

Reply:
Yep, I did mean `:code:`, thanks!

But I'll update all the ``` `:code:X` ``` in this file to ``` `X` ```

Reply:
Done in 89c589639c1f1f318f4ff427bcb9fea4d99c1515.